### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java [hunk 1]
```diff
@@ -86,16 +89,22 @@ public void contextInitialized(ServletContextEvent event) {
...
-    PropertyConfigurator.configure(application.getRealPath("/") + "/WEB-INF/log4j.miso.properties");
-
     XmlWebApplicationContext context = (XmlWebApplicationContext) WebApplicationContextUtils.getRequiredWebApplicationContext(application);
 
     // resolve property file configuration placeholders
     MisoPropertyExporter exporter = (MisoPropertyExporter) context.getBean("propertyConfigurer");
     Map<String, String> misoProperties = exporter.getResolvedProperties();
 
+    // load logging system manually so that we can inject the baseDirectory property
+    ThreadContext.put("baseDirectory", misoProperties.get("miso.baseDirectory"));
```
Comment (apmasell) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java:99]: Why is this going in a thread local?

**Review by apmasell — APPROVED** [review]

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java (hunk 1 above)
Comment (djcooke) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java:99]: log4j2 has several methods of [property lookup](https://logging.apache.org/log4j/2.x/manual/lookups.html). It wasn't my first choice, but ThreadContext map is the one I got to work. Spring lookup seems to only work for Spring Boot.

Comment (djcooke) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java:99]: Actually, I could set a system property and use that instead. Probably makes more sense

Comment (apmasell) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java:99]: Worth putting a comment about it too.

Comment (djcooke) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java:99]: Beyond the existing comment about injecting the property?

Comment (apmasell) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java]: Yeah. It's unclear that the property files are the consumers.

Comment (djcooke) [reply miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java]: Updated

**Review by avarsava — APPROVED** [review]

**Pull request merged**